packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -441,7 +441,8 @@ class DevFS {
     _entries.forEach((Uri deviceUri, DevFSContent content) {
       if (!content._exists) {
         final Future<Map<String, dynamic>> operation =
-            _operations.deleteFile(fsName, deviceUri);
+            _operations.deleteFile(fsName, deviceUri)
+            .then((dynamic v) => v?.cast<String,dynamic>());
```

Comment:
What does the 'dynamic' return type of deleteFile mean, what is the meaning of expected type of String->dynamic mean? Does this cast always hold?

Reply:
+1 - is it coming in as `Map<dynamic, dynamic>`?  Can we specify so instead of `dynamic`?

Reply:
The return type for `deleteFile` is `dynamic`, nothing more specific. Looking at the implementations of the api, the cast does hold, return type is either `Null` or `Map`.